app/ios/scripts/extract_dart_defines.sh

```
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# Dart defineを書き出すファイルパスを指定します。
+# ここでは `Environment.xcconfig` というファイル名で作成することにします。
+OUTPUT_FILE="${SRCROOT}/Flutter/DartDefines.xcconfig"
+# Dart defineの中身を変更した時に古いプロパティが残らないように、初めにファイルを空にしています。
+: > $OUTPUT_FILE
+
+# この関数でDart defineをデコードします。
+function decode_url() { echo "${*}" | base64 --decode; }
```

Shellcheck (0.10.0)

[warning] 'function' keyword is non-standard. Delete it.

(SC2112)